### ISSUE TITLE
[doc] Issue in "Kubernetes folder"

### ISSUE THREAD

**Issue by peteingithub** [suse-edge/suse-edge.github.io#273]
Kubernetes folder:

https://suse-edge.github.io/atip-management-cluster.html#mgmt-cluster-kubernetes-folder

Is the setting `installCRDs: "true"` for cert-manager recommended?

**Comment by e-minguez** [comment]
You can install cert-manager certs using two ways, see https://cert-manager.io/docs/installation/helm/#3-install-customresourcedefinitions
Both have pros and cons, see https://cert-manager.io/docs/installation/helm/#crd-considerations

**Comment by peteingithub** [comment]
Yeah, I'm just wondering if that could lead to un-install of all the certs in a cluster accintally. Thus asking if that feature is _recommended_.

**Comment by ranjinimn** [comment]
@e-minguez What is the recommended method to install cert-manager?
